helm/idol-answerserver/templates/configmap-prestart.yaml

```
@@ -0,0 +1,20 @@
+# BEGIN COPYRIGHT NOTICE
+# Copyright 2024 Open Text.
+# 
+# The only warranties for products and services of Open Text and its affiliates and licensors
+# ("Open Text") are as may be set forth in the express warranty statements accompanying such
+# products and services. Nothing herein should be construed as constituting an additional warranty.
+# Open Text shall not be liable for technical or editorial errors or omissions contained herein.
+# The information contained herein is subject to change without notice.
+#
+# END COPYRIGHT NOTICE
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: idol-answerserver-cfg
```

```suggestion
  name: idol-answerserver-scripts
```
Possibly less confusing name? Has knock-on effects to values.yaml